### ISSUE TITLE
6-Attribute-Centric Compositional Text-to-Image Generation

### ISSUE THREAD

**Issue by mjmaher987** [sharif-ml-lab/visual_compositional_generation_survey#6]
## Task: Read the Paper and Do the following 4 steps
1. Highlight the paper systematically (**green** for important notes - **blue** for very important notes that worth mentioning in the final survey - **red** for the sentences that we don’t understand or need further attention in the future) [In Zotero].
2. Write the Wikipedia summary of the paper in the GitHub issue (_comment in this issue_).
3. Extract the information in the [excel file](https://docs.google.com/spreadsheets/d/15_sh6TqtUvZIJo1-RZxQtsG0Nvg0vPG_IFKCWiGMjdY/edit?usp=sharing).
4. Add the related sentences to the [overleaf file](https://www.overleaf.com/8167246262ccxtrzztjvpz). Note that when you add the sentence, don’t forget to write the reference in the format of ”[ #reference number that is mentioned in the Google Doc - Name of the Author (who added this to the overleaf) ]” at the end of the sentence.

**Comment by shafieiali42** [comment]
### **Introduction**
generative models have difficulty in capturing the data distribution of underrepresented attribute compositions while over-memorizing overrepresented attribute compositions, which raises public concerns about their robustness and fairness. This paper proposes ACTIG, an attribute-centric compositional text-to-image generation framework to tackle this challenge.

### **Contributions**

• They presented an attribute-centric compositional text-to-image generation framework, named ACTIG, which excels in image quality and text-image consistency.
• They alternated between a fully supervised paradigm and a novel image-free paradigm in training so that the model learns the feature distributions of the real data and underrepresented attribute compositions from attribute-centric feature augmentation simultaneously.
• They proposed an attribute-centric contrastive loss to disentangle the data distribution of attributes to prevent the model from over-memorizing the overrepresented attribute compositions.
• They conducted comprehensive experiments on the CelebAHQ dataset and CUB dataset, and ACTIG achieved state-of-the-art results.

### **Methodology**
**Text-to-image mapping** : 
We propose a text-to-image mapping network that projects a CLIP text encoding t into CLIP image space to obtain the approximate CLIP image encoding i  ̃, which is later used for image-free training.

![mapping network](https://github.com/sharif-ml-lab/visual_compositional_generation_survey/assets/62015693/f99eee91-b861-447f-bba4-ce85fbf0bc5c)

**Matching discriminator **:

![matching discriminator](https://github.com/sharif-ml-lab/visual_compositional_generation_survey/assets/62015693/396131de-8e48-4ddd-b909-3cc4490773ff)

**Attribute-centric feature augmentation**: 
They generated  the text with underrepresented attribute compositions by randomly selecting attributes to form prompts (for CelebA-HQ) or by replacing attributes in the training prompts with other randomly sampled attributes (for CUB)

**Attribute-centric contrastive loss**:
To prevent overfitting to overrepresented attribute compositions and disentangling their distributions, they proposed an attribute-centric contrastive loss that leverages the generality and transferability of CLIP.

**Training schemes**:
![algorithm](https://github.com/sharif-ml-lab/visual_compositional_generation_survey/assets/62015693/1c535230-220e-48c8-b2d5-43c6dc08906e)
